generators/connection/index.js

```diff
@@ -243,6 +244,20 @@ module.exports = class ConnectionGenerator extends Generator {
             return false;
           }
 
+          if(typeof connectionString === 'object' && database === 'rethinkdb') {
```

Comment:
Nope, lines starting at 55 take care of creating a _new_ connection object, but that's useless during the prompts.

The `connectionString` pompt's `when` function checks to see if a connection is already defined in the configuration and skips the question if it exists. 
Right now it checks for a "string" and returns that string if it exists, but RethinkDB is an object.

My solution re-creates a connection string by destructuring the connection object, thus not tearing apart the way it currently works while keeping the already defined connection settings.

Does that make sense?